lib/taiko.js

```diff
@@ -1439,16 +1439,27 @@ module.exports.below = selector => {
  *
  * @param {selector|string} selector - Web element selector.
  * @returns {RelativeSearchElement}
+ *
```

Comment:
this breaks the docs, try `npm run doc-new`
I feel description can be updated better to convey this. worth mentioning the offset

Reply:
The fix is available in [commit](https://github.com/getgauge/taiko/pull/302/commits/55c404d37f04d771dcc0a0f71d2af7aa388890ae)